README.md

```diff
@@ -44,25 +44,120 @@ your operating system from the
+- Drag the orca icon into the `Applications` folder.
+- Open finder and navigate to the `Applications/` folder.
+- Right-click on the orca icon and select _Open_ from the context menu.
+- A password dialog will appear asking for permission to add orca to your system `PATH`.
+- Enter you password and click _OK_.
+- This should open an _Installation Succeeded_ window.
+- Open a new terminal and verify that the orca executable is available on your `PATH`. 
+
+```
+$ which orca
+/usr/local/bin/orca
+
+$ orca --help
+Plotly's image-exporting utilities
+
+  Usage: orca [--version] [--help] <command> [<args>]
+  ...
+```
 
 #### Windows
 
-- Extract the `window-release.zip` file
-- In the `release` folder, double-click on `orca Setup X.Y.Z`, this will create an orca icon on your Desktop
-- Right-click on the orca icon, then click on _Properties_ and copy the _Starts in_ field
-- In the command prompt, run `PATH %PATH%;<paste the "Starts in" field here>`
+- Extract the `windows-release.zip` file.
+- In the `release` folder, double-click on `orca Setup X.Y.Z`, this will create an orca icon on your Desktop.
+- Right-click on the orca icon and select _Properties_ from the context menu.
+- From the _Shortcut_ tab, copy the directory in the _Start in_ field.
+- Add this _Start in_ directory to you system `PATH` (see below).
+- Open a new Command Prompt and verify that the orca executable is available on your `PATH`.
+
+```
+> orca --help
+Plotly's image-exporting utilities
+
+  Usage: orca [--version] [--help] <command> [<args>]
+  ...
+```
+
+##### Windows References
+ - How to set the path and environment variables in Windows: https://www.computerhope.com/issues/ch000549.htm
 
 #### Linux
 
-- Run `$ chmod +x orca-X.Y.Z-x86_64.AppImage`
-- Add it to your `$PATH`
+- Make the orca AppImage executable.
+
+```
+$ chmod +x orca-X.Y.Z-x86_64.AppImage
+```
+
+- Create a symbolic link named `orca` somewhere on your `PATH` that points
+to the AppImage.
+
+```
+$ ln -s /path/to/orca-X.Y.Z-x86_64.AppImage /somewhere/on/PATH/orca
+```
+
+- Open a new terminal and verify that the orca executable is available on your `PATH`. 
+
+```
+$ which orca
+/somewhere/on/PATH/orca
+
+$ orca --help
+Plotly's image-exporting utilities
+
+  Usage: orca [--version] [--help] <command> [<args>]
+  ...
+```
+ 
+##### Linux Troubleshooting: Cannot open shared object
+The Electron runtime depends a several common system libraries. These
+libraries are pre-installed in most desktop Linux distributions
+(e.g. Ubuntu), but are not pre-installed on some server Linux distributions
+(e.g. Ubuntu Server). If a shared library is missing, you will see an error
+message like:
+
+```
+$ orca --help
+orca: error while loading shared libraries: libgtk-x11-2.0.so.0:
+cannot open shared object file: No such file or directory
+```
+
+On Ubuntu Server, these additional libraries can be installed with:
+
+```
+sudo apt-get install libgtk2.0-0 libxtst6 libxss1 libgconf-2-4 libnss3 libasound2
```

Comment:
Could you instead tell users to install `google-chrome` (like in our image-server [Dockerfile](https://github.com/plotly/orca/blob/4bd21148fe4a1f13ee783b20b26fbf8788186583/deployment/Dockerfile#L90-L94)? That and `libgtk2.0-0` / `libgconf-2-4` should be enough to run orca smoothly

Reply:
I would prefer if the installation instructions match something that we can test.

Pointing users to the what we use in our image-server container should achieve that (as we test it on every push and on prod). But, perhaps our Dockerfile is a little too Ubuntu-centric? Do you think many people with try to user orca in non-Ubuntu server environments?

Reply:
Sounds good. Done in a87754d 

Let me know if you think of anything else!